embulk-core/src/main/java/org/embulk/cli/EmbulkRun.java

```diff
@@ -235,20 +235,39 @@ private int runBundler(final List<String> arguments, final Path path, final Embu
         localJRubyContainer.runScriptlet("require 'bundler/friendly_errors'");
         localJRubyContainer.runScriptlet("require 'bundler/cli'");
 
+        localJRubyContainer.runScriptlet("__internal_exit_status__ = 0");
         localJRubyContainer.put("__internal_argv_java__", arguments);
         if (path == null) {
-            localJRubyContainer.runScriptlet("Bundler.with_friendly_errors { Bundler::CLI.start(Array.new(__internal_argv_java__), debug: true) }");
+            localJRubyContainer.runScriptlet(
+                    "begin;"
+                    + "  Bundler.with_friendly_errors { Bundler::CLI.start(Array.new(__internal_argv_java__), debug: true) }"
```

Comment:
missing `;`? 

```
embulk-0.10.48-SNAPSHOT.jar bundle hoge

...
Exception in thread "main" org.jruby.embed.ParseFailedException: (SyntaxError) <script>:1: syntax error, unexpected =>
... true) }  rescue SystemExit => __internal_system_exit__;    ...
                                                                                                                           ^~
	at org.jruby.embed.internal.EmbedRubyRuntimeAdapterImpl.runParser(EmbedRubyRuntimeAdapterImpl.java:199)
	at org.jruby.embed.internal.EmbedRubyRuntimeAdapterImpl.parse(EmbedRubyRuntimeAdapterImpl.java:104)
	at org.jruby.embed.ScriptingContainer.parse(ScriptingContainer.java:1228)
	at org.jruby.embed.ScriptingContainer.runScriptlet(ScriptingContainer.java:1288)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.embulk.jruby.ScriptingContainerDelegateImpl.runScriptlet(ScriptingContainerDelegateImpl.java:841)
	at org.embulk.jruby.LazyScriptingContainerDelegate.runScriptlet(LazyScriptingContainerDelegate.java:305)
	at org.embulk.cli.EmbulkRun.runBundler(EmbulkRun.java:241)
	at org.embulk.cli.EmbulkRun.runInternal(EmbulkRun.java:64)
	at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:24)
	at org.embulk.cli.Main.main(Main.java:53)
Caused by: org.jruby.exceptions.SyntaxError: (SyntaxError) <script>:1: syntax error, unexpected =>
... true) }  rescue SystemExit => __internal_system_exit__;    ...
```

Reply:
Thanks for your good catch! Fixed, with some more updates.